dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/TrackerTrackedEntityCriteriaMapperTest.java

```diff
@@ -215,7 +216,7 @@ void testMapping() throws BadRequestException, ForbiddenException {
     criteria.setSkipPaging(false);
     criteria.setIncludeDeleted(true);
     criteria.setIncludeAllAttributes(true);
-    criteria.setOrder(Collections.singletonList(OrderCriteria.of("created", SortDirection.ASC)));
+    criteria.setOrder(Collections.singletonList(OrderCriteria.of("createdAt", SortDirection.ASC)));
 
```

Comment:
Do we have MVC tests where we are testing the 400 response?

Reply:
Covered by test `testMappingOrderParamsGivenInvalidField` below IMHO